stylesheets/pages/products_list.css

```diff
@@ -0,0 +1,82 @@
+.as-aside {
+  width: 15rem;
+  padding-top: var(--sp-sm);
+  box-shadow: 0 0 2px var(--gray-500);
```

Comment:
Suggestion: If you will use this box shadow property many times, then directly create a variable for `box-shadow-1` in your variable.css.